### PR TITLE
feature: badges v2 and input

### PR DESCRIPTION
**Pull request by saiyerniakhil**
--
* add avatars 👤 with number badges🥇 component
* add icons with number badges and dot badges 🔴 
* add input fields with errors ❌ ✔️ and validations - text box, text area
* add assets required for form inputs
* **TODO**: padding and spacing are yet to be added as utility classes.
* screenshots -

brave_5yaqxwjR5F

brave_mZyWNOjkDi

input fields in action -
brave_PZkuOdP1L2
brave_JHfDeeyTO0

### CODE REVIEW

File: styles/components/avatar.css [hunk 1]
```diff
@@ -44,3 +76,33 @@
...
+  border-radius: 50%;
+  width: 18%;
+  height: 18%;
+}
+.badge--dot {
+  width: 25%;
+  height: 25%;
+}
+.icon > svg {
+  width: 32px;
```
Comment (lannisterrr) [inline styles/components/avatar.css:98]: you can use rems here

File: styles/components/avatar.css [hunk 2]
```diff
@@ -44,3 +76,33 @@
...
+  position: absolute;
+  border-radius: 50%;
+  width: 18%;
+  height: 18%;
+}
+.badge--dot {
+  width: 25%;
+  height: 25%;
+}
+.icon > svg {
```
Comment (lannisterrr) [inline styles/components/avatar.css:97]: you can also give a specific class to SVG instead of targeting it directly.

**Review by lannisterrr — COMMENTED** [review]
I mentioned my suggestions

**Review by Ronak-Mutha — COMMENTED** [review]
Code looks clean and good to merge. Just a suggestion to use pre defined variables for smaller spacing sizes.

File: styles/components/avatar.css (hunk 2 above)
Comment (saiyerniakhil) [reply styles/components/avatar.css:97]: yes but I'm using a third party icon library so I don't have any option to add class name to svgs

File: styles/components/avatar.css (hunk 1 above)
Comment (saiyerniakhil) [reply styles/components/avatar.css:98]: I will make those changes in new PR.

**Pull request merged**